components/MobileNavigation.js

```diff
@@ -164,8 +192,9 @@ function NavigationLink ({ href, disabled, children }) {
           opacity: 0.5;
         }
         .disabled {
-          opacity: 0.25;
-          pointer-events: none;
+          /* opacity: 0.25;
+          pointer-events: none; */
```

Comment:
Don’t hack to change disabled into active state. Please create a new class `.active` to represent the active state.